src/constants.js

```diff
@@ -0,0 +1,7 @@
+import path from 'path';
+
+export const SOURCE_URL = 'https://github.com/edx/front-end-cookie-cutter-application/archive/master.zip';
```

Comment:
Style nit: I prefer single export when there are multiple things to be exported i.e

```javascript
export {
  SOURCE_URL,
  TEMP_DIR,
  ...
}
```

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md#importprefer-default-export